dealer.rb

```diff
@@ -0,0 +1,13 @@
+$LOAD_PATH << '.'
```

Comment:
Rather than manipulating the LOAD_PATH, ruby now provides a `require_relative` statement which would allow you to rewrite these like

```ruby
require_relative 'player'
```